src/index.ts

```diff
@@ -76,6 +79,8 @@ module.exports = class CustomWorkerService {
   async before(capabilities, specs, browser) {
     // Errors in WDIO hooks are suppressed by default => we call process.exit(1). It will mark all specs as failed
     try {
+      browser.config = await getValue("config");
+      browser.params = browser.config.params;
```

Comment:
Is there a reason to do this? It works for me with qmate-service code for v8, without these 2 lines, and the setValue call in beforeSession hook

Reply:
Sorry used the wrong runner - 7.x version. 
Only thing is @wdio/shared-store-service should be moved from devDependencies to dependencies. Otherwise service doesn't load outside of this project.